src/proj_constants.h

```diff
@@ -513,6 +513,9 @@
 #define EPSG_CODE_METHOD_VERTCON 9658
 #define EPSG_NAME_METHOD_VERTCON "VERTCON"
 
+#define EPSG_CODE_METHOD_VERTICALGRID_NZLVD 1071
+#define EPSG_NAME_METHOD_VERTICALGRID_NZLVD "NZLVD"
```

Comment:
The value should be the exact name in EPSG, so "Vertical Offset by Grid Interpolation (NZLVD)"